docker-compose-generator/docker-fragments/opt-add-lightning-terminal.yml

```diff
@@ -4,7 +4,7 @@ services:
     environment:
       BTCPAY_EXTERNALSERVICES: "Lightning Terminal:${BTCPAY_PROTOCOL:-https}://${BTCPAY_HOST}/lit/;"
   lnd_lit:
-    image: "lightninglabs/lightning-terminal:v0.6.1-alpha-path-prefix"
+    image: "lightninglabs/lightning-terminal:v0.6.2-alpha-path-prefix"
```

Comment:
If we're updating this file, shouldn't we also make sure that all of LiT's data is written to the volume correctly? I see there is a volume at `/data/lit`, but LiT isn't configured to use it. Either an ENV var `HOME=/data/lit` should be set (or is this done somewhere else?) or the volume should be mounted to `/root`. See https://github.com/btcpayserver/btcpayserver-docker/issues/592. Currently Pool's data (and I assume Loop, Faraday and Lit's internal data) is lost when the container is re-created.

Reply:
Thanks for the pointer - ok like [cfaed5f](https://github.com/btcpayserver/btcpayserver-docker/pull/602/commits/cfaed5fa23c0ae384b20c5c661c781e7fde6d175)?

Reply:
Looks good, yes. Thanks! I'm not able to test this though.